### PR TITLE
added newfile

### PR DESCRIPTION
**Pull request by georgeynrcs**
I've added newfile. Please review my code.

### CODE REVIEW

**Comment by albertgeorge56** [comment]
add content also

**Comment by georgeynrcs** [comment]
added the content too.

File: newfile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+new content
```
Comment (albertgeorge56) [inline newfile.txt:1]: delete this one and add new

Comment (georgeynrcs) [reply newfile.txt]: deleted and added new

Comment (georgeynrcs) [reply newfile.txt]: got issue

Comment (georgeynrcs) [reply newfile.txt]: no

**Pull request merged**